lib/daisy_ui_components/tabs.ex

```diff
@@ -0,0 +1,399 @@
+
+  def tab(%{type: "label", title: _title} = assigns) do
+    assigns =
+      assign(assigns, :class, tab_classes(assigns))
+
+    ~H"""
+    <.tab_label title={@title} name={@name} class={@class} active={@active} {@rest} />
+    """
+  end
+
+  def tab(%{type: "label"} = assigns) do
+    assigns =
+      assign(assigns, :class, tab_classes(assigns))
+
+    ~H"""
+    <.tab_label name={@name} class={@class} active={@active} {@rest}>
+      {render_slot(@inner_block)}
+    </.tab_label>
+    """
+  end
+
+  def tab(%{type: "radio", title: _title} = assigns) do
+    assigns =
+      assign(assigns, :class, radio_tab_classes(assigns))
+
+    ~H"""
+    <.tab_radio title={@title} name={@name} class={@class} active={@active} {@rest} />
+    """
+  end
+
+  def tab(%{type: "radio"} = _assigns) do
+    raise ArgumentError,
+          "title attribute is required for radio type tab. Radio type tab cannot have inner block content"
+  end
+
+  def tab(%{type: "button", title: _title} = assigns) do
+    assigns =
+      assign(assigns, :class, tab_classes(assigns))
+
+    ~H"""
+    <.tab_button title={@title} class={@class} {@rest} />
+    """
+  end
+
+  def tab(%{type: "button"} = assigns) do
+    assigns =
+      assign(assigns, :class, tab_classes(assigns))
+
+    ~H"""
+    <.tab_button class={@class} {@rest}>
+      {render_slot(@inner_block)}
+    </.tab_button>
+    """
+  end
+
+  def tab(%{title: _title} = assigns) do
+    assigns =
+      assign(assigns, :class, tab_classes(assigns))
+
+    ~H"""
+    <.tab_link title={@title} class={@class} {@rest} />
+    """
+  end
+
+  def tab(assigns) do
+    assigns =
+      assign(assigns, :class, tab_classes(assigns))
+
+    ~H"""
+    <.tab_link class={@class} {@rest}>
+      {render_slot(@inner_block)}
+    </.tab_link>
+    """
+  end
+
+  attr :title, :string
+  attr :name, :string, required: true
+  attr :class, :any, default: nil
+  attr :active, :boolean, default: false
+  attr :rest, :global
+
+  slot :inner_block
+
+  def tab_label(%{title: _title} = assigns) do
+    ~H"""
+    <label class={@class} {@rest}>
+      <input type="radio" name={@name} checked={@active && "checked"} />
+      {@title}
+    </label>
+    """
+  end
+
+  def tab_label(assigns) do
+    ~H"""
+    <label class={@class} {@rest}>
+      <input type="radio" name={@name} checked={@active && "checked"} />
+      {render_slot(@inner_block)}
+    </label>
+    """
+  end
+
+  attr :title, :string, required: true
+  attr :name, :string, required: true
+  attr :class, :any, default: nil
+  attr :active, :boolean, default: false
+  attr :rest, :global
+
+  def tab_radio(assigns) do
+    ~H"""
+    <input
+      type="radio"
+      name={@name}
+      class={@class}
+      aria-label={@title}
+      checked={@active && "checked"}
+      {@rest}
+    />
+    """
+  end
+
+  attr :title, :string
+  attr :class, :any, default: nil
+  attr :rest, :global
+
+  slot :inner_block
+
+  def tab_button(%{title: _title} = assigns) do
```

Comment:
I think this one is not need it right? I could not find tab button on the daisyUI examples